packages/tools/cli/src/index.ts

```diff
@@ -1,140 +1,6 @@
-          const packagesNotCoveredByNolyfill = await findPackagesNotCoveredByNolyfill(packageManager, projectPath);
-
-          if (packagesNotCoveredByNolyfill.length === 0) {
-            console.log(`${picocolors.green('Congratulations! Your project does not contain any redundant polyfills that can be optimized by nolyfill 🚀')}\n`);
-          } else {
-            await generateIssue(projectPath, packageManager, packagesNotCoveredByNolyfill);
-          }
-        } else {
-          console.log(picocolors.yellow(`Found ${picocolors.green(picocolors.bold(packagesToBeOverride.length))} redundant packages:`));
-          console.log(renderTree(packagesToBeOverride));
-
-          console.log(`Run "${picocolors.bold(picocolors.green('nolyfill install'))}" to replace them with a super lightweight ✨ version.\n`);
-        }
-      });
-
-    program
-      .command('install')
-      .description('replace redundant polyfills inside your project with nolyfill')
-      .argument('[path]', 'project path to install nolyfill into')
-      .addOption(pmCommandOption)
-      .action(async (source: string | undefined, option: PmCommandOptions) => {
-        const projectPath = path.resolve(source ?? process.cwd());
-        const packageManager = option.pm === 'auto' ? await detectPackageManager(projectPath) : option.pm;
-
-        if (checkUnsupportedPM(packageManager)) {
-          return;
-        }
-
-        const packagesToBeOverride = await findPackagesCoveredByNolyfill(packageManager, projectPath);
-
-        if (packagesToBeOverride.length === 0) {
-          const packagesNotCoveredByNolyfill = await findPackagesNotCoveredByNolyfill(packageManager, projectPath);
-
-          if (packagesNotCoveredByNolyfill.length === 0) {
-            console.log(`${picocolors.green('Congratulations! Your project does not contain any redundant polyfills that can be optimized by nolyfill 🚀')}\n`);
-          } else {
-            await generateIssue(projectPath, packageManager, packagesNotCoveredByNolyfill);
-          }
-        } else {
-          console.log(picocolors.yellow(`Found ${picocolors.green(picocolors.bold(packagesToBeOverride.length))} redundant packages:`));
-          console.log(renderTree(packagesToBeOverride));
-
-          await overridesPackageJson(packageManager, projectPath, packagesToBeOverride);
-
-          printPostInstallInstructions(packageManager);
-        }
-      });
-
-    await checkForUpdates('nolyfill', version);
-    await program.parseAsync(process.argv);
-  } catch (e) {
-    handleError(e as Error, !!program.opts<CliOptions>().debug);
-  }
-})();
+// Programmatic APIs
+export { allPackages } from './all-packages';
+export { findPackagesCoveredByNolyfill, findPackagesNotCoveredByNolyfill } from './find-coverable-packages';
+export { overridesPackageJson } from './lib/json';
+export { detectPackageManager, type PackageManager } from './package-manager';
+export * from './types';
```

Comment:
The diff is a bit off 😅

Basically what happened is that I renamed the `index.ts` to `cli.ts`, and have the new `index.ts` for JS APIs. This way we could have CLI and programmatic usage within the same package.

You might want to revise which APIs you think that makes sense to expose here.